src/components/MultiSelect/index.js

```diff
@@ -10,6 +10,7 @@ export const MultiSelect = React.forwardRef(({
   options,
   value,
   onChange,
+  onBlur,
```

Comment:
I think we can add all possible event props from Prime for MultiSelect https://primereact.org/multiselect/#api.MultiSelect.callbacks and for Dropdown
https://primereact.org/dropdown/#api.Dropdown.callbacks